packages/ui5-middleware-websocket/ui5.yaml

```diff
@@ -0,0 +1,16 @@
+# https://sap.github.io/ui5-tooling/pages/extensibility/CustomServerMiddleware/
+specVersion: "3.0"
+metadata:
+  name: ui5-middleware-websocket
+kind: extension
+type: server-middleware
+middleware:
+  path: lib/index.js
+---
+specVersion: "3.0"
+metadata:
+  name: ui5-middleware-websocket-test
+kind: extension
+type: server-middleware
+middleware:
+  path: lib/test.js
```

Comment:
This exposes the `ui5-middleware-websocket-test` middleware for consumers of the package, right?
What about moving it to another file?